### PR TITLE
feat: update test for method with clientid

### PR DESCRIPTION
**Pull request by infantesimone**
#### Motivation and Context

The goal of this PR is to add api test for payment method with clientId.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by Fernando-Granato — DISMISSED** [review]
OK just a consideration we could do a GET payment-method/:id for IO.
Anyway, it's OK with me

**Comment by infantesimone** [comment]
> OK just a consideration we could do a GET payment-method/:id for IO. Anyway, it's OK with me

add new test for IO @Fernando-Granato

File: api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json [hunk 1]
```diff
@@ -28,7 +27,7 @@
 							"  const responseJson = pm.response.json();",
 							"  pm.expect(responseJson.id).to.be.a('string');",
 							"  pm.expect(responseJson.id).to.have.length.greaterThan(0);",
-							"  pm.environment.set(\"PAYMENT_METHOD_ID\", responseJson.id);",
+							"  pm.environment.set(\"PAYMENT_METHOD_ID_IO\", responseJson.id);",
```
Comment (Fernando-Granato) [inline api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json:30]: Maybe the env file needs to be modified?

File: api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json [hunk 2]
```diff
@@ -79,15 +131,139 @@
...
+					}
+				],
+				"url": {
+					"raw": "{{HOSTNAME}}/payment-methods/{{PAYMENT_METHOD_ID_IO}}",
+					"host": [
+						"{{HOSTNAME}}"
+					],
+					"path": [
+						"payment-methods",
+						"{{PAYMENT_METHOD_ID_IO}}"
```
Comment (Fernando-Granato) [inline api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json:266]: Maybe the env file needs to be modified?

File: api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json [hunk 3]
```diff
@@ -79,15 +131,139 @@
...
+				"method": "GET",
+				"header": [
+					{
+						"key": "x-client-id",
+						"value": "CHECKOUT",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{HOSTNAME}}/payment-methods/{{PAYMENT_METHOD_ID_CHECKOUT}}",
```
Comment (Fernando-Granato) [inline api-tests/payment-method-service/payment-method-service_ecommerce_local.postman_collection.json:220]: Maybe the env file needs to be modified?

**Review by Fernando-Granato — COMMENTED** [review]
You should not modify the env file with the new variables

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**